### PR TITLE
New theorem: Pseudometrizable spaces are perfectly normal

### PR DESCRIPTION
**Pull request by prabau**
This handles half of issue #90.

Optionally, if you want to, we could add a reference to J. Kelley, General Topology, as it is stated explicitly in Exercise K, p. 134 in that book ("Each pseudometrizable space is perfectly normal").  You'd have to get the MR number for it (I don't have access to MR).

@StevenClontz

### CODE REVIEW

**Comment by ccaruvana** [comment]
@prabau I agree with adding this theorem. If you want to include the MR number before approval, it is MR0370454

**Comment by StevenClontz** [comment]
BTW there's a public lookup interface: https://mathscinet.ams.org/mrlookup You can't see the reviews, but that'll get you MR numbers.

**Comment by prabau** [comment]
> 
> 
> BTW there's a public lookup interface: https://mathscinet.ams.org/mrlookup You can't see the reviews, but that'll get you MR numbers.

Thanks.  That's very useful.  I'll add the reference.

**Review by ccaruvana — APPROVED** [review]

**Pull request merged**
